MantidQt/CustomInterfaces/src/Indirect/ISISCalibration.cpp

```diff
@@ -714,6 +698,32 @@ void ISISCalibration::pbRunFinished() {
 
   m_uiForm.leRunNo->setEnabled(true);
 }
+/**
+ * Handle saving of workspace
+ */
+void ISISCalibration::saveClicked() {
+  checkADSForPlotSaveWorkspace(m_outputCalibrationName.toStdString(), false);
+  addSaveWorkspaceToQueue(m_outputCalibrationName);
+  checkADSForPlotSaveWorkspace(m_outputResolutionName.toStdString(), false);
+  addSaveWorkspaceToQueue(m_outputResolutionName);
+  m_batchAlgoRunner->executeBatchAsync();
+}
+
+/**
+ * Handle mantid plotting
+ */
+void ISISCalibration::plotClicked() {
 
+  plotTimeBin(m_outputCalibrationName);
+  checkADSForPlotSaveWorkspace(m_outputCalibrationName.toStdString(), true);
+  QStringList plotWorkspaces;
+  if (m_uiForm.ckCreateResolution->isChecked()) {
+    checkADSForPlotSaveWorkspace(m_outputResolutionName.toStdString(), true);
+    plotWorkspaces << m_outputResolutionName;
```

Comment:
Could we change this to be `plotWorkspaces.append(m_outputResolutionName)` as it makes it explicit what the `<<` operator is doing